examples/test_uds/test_uds_server.py

```diff
@@ -0,0 +1,54 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+UDS server auxiliary simple example
+***********************************
+
+:module: test_uds_server
+
+:synopsis: Example test that shows how to use the UDS server auxiliary
+
+.. currentmodule:: test_uds_server
+
+"""
+
+import logging
+import time
+import unittest
+
+import pykiso
+from pykiso.auxiliaries import uds_aux
+
+
+@pykiso.define_test_parameters(suite_id=1, case_id=1, aux_list=[uds_aux])
+class ExampleUdsServerTest(pykiso.BasicTest):
+    def setUp(self):
+        """Hook method from unittest in order to execute code before test case run."""
+        uds_aux.register_callback(0x1003, 0x50031234)
```

Comment:
```suggestion
        uds_aux.register_callback(request = 0x1003, response =  0x50031234)
```

Reply:
Should be good to go now, test example and docs have been extended as much as possible